template/php7/function/src/Handler.php

```diff
@@ -9,10 +9,10 @@
 class Handler
 {
     /**
-     * @param $data
-     * @return
+     * @param string $data
+     * @return string
      */
-    public function handle($data)
+    public function handle(string $data): string
```

Comment:
Before you made this explicitly into a string, what kind of data was being passed in here?